report/01_introduction/00_introduction.md

```diff
@@ -0,0 +1,14 @@
+## Introduction
+
+Cognitive fatigue arises from prolonged periods of time in which mentally demanding tasks are performed that induce a state of subjective exhaustion, feelings of “tiredness and lack of energy”, and decline in performance [1]. It can be said that mental fatigue is felt much more often after mentally strenuous tasks than after physical exertion. Examples that can lead to cognitive fatigue are, for example, strenuous work days without appropriate compensation or processing the tax return [2].
+Especially with prolonged mental stress, we often feel drained and have the feeling that we can no longer continue. A classic example of this is a long drive. This strains our brain so much that after a certain time we are no longer able to drive and need a break, although the body itself did not have to work hard [2]. Symptoms of cognitive fatigue can mainly be described as follows: poor concentration, forgetfulness, slight irritation, simple tasks seem difficult to solve, feeling overwhelmed, physical stress symptoms such as headaches, tension and an increased heart rate, feeling of a mental block, you feel foggy and sleep is harder [3].
+
+Since there is currently no standardized test for recording cognitive fatigue available and biological markers cannot be used to identify these [4], the task was to write an application that should determine cognitive fatigue as part of a study on the basis of daily self-tests.
+
+The application should therefore be able to determine the general condition of a subject with the help of a questionnaire that should be filled out regularly and replace earlier paper questionnaires and be able to save it centrally in a database. In addition, a self-test should be available, which in the best case should be completed several times a day. This should contain two tasks that check the respondent's reaction time and memory ability. In addition, with each run of the self-test, the current emotional state and the current activity of the test person are recorded so that conclusions can be drawn from them later. This information is recorded in a database that is to be exchanged regularly with a server database in order to be able to export all user data centrally for a study.
+
+
+[1] Shortz, A.E., Pickens, A., Zheng, Q. et al. The effect of cognitive fatigue on prefrontal cortex correlates of neuromuscular fatigue in older women. J NeuroEngineering Rehabil 12, 115 (2015). https://doi-org.ezproxy.hs-augsburg.de/10.1186/s12984-015-0108-3

```

Comment:
I'd put sources in a separate file at the end of the report 🤔

Reply:
Yes, the seperate file is already there, but in order not to get confused, it's hard to find unambiguous names for the references. I think when we're working on it at the same time, it's difficult to set it in a global place. My suggestion would be to first put the references in the respective file and at the end someone looks over it and summarizes it and sets the numbering?